addon/ng2/utilities/route-utils.ts

```diff
@@ -70,6 +70,9 @@ export function bootstrapItem(mainFile, imports: {[key: string]: [string, boolea
 
 export function insertImport(fileToEdit: string, symbolName: string,
                                   fileName: string, isDefault = false): Change {
+        
+  fileName = fileName.replace(/\\/, '/'); // correction in windows           
```

Comment:
This probably means that the generation of the `'./aa\bb/bb.component'` path is the real issue. Check invocation of `insertImport` and see how the third argument `fileName` is being composed.